coverart_redirect/request.py

```diff
@@ -346,6 +350,8 @@ def handle(self, request):
             return self.handle_index()
         if entity == 'robots.txt':
             return self.handle_robots()
+        if entity == 'api':
+            return self.handle_api()
```

Comment:
You'll need to pass the `request` variable here and adjust the signature of `handle_api`, but should work after that.